src/Common/src/TypeSystem/Ecma/EcmaSignatureParser.cs

```diff
@@ -16,13 +18,20 @@ public struct EcmaSignatureParser
         private EcmaModule _module;
         private BlobReader _reader;
 
-        // TODO
-        // bool _hasModifiers;
+#if TYPE_LOADER_IMPLEMENTATION
+        private LowLevelStack<int> _indexStack;
+#else
+        private Stack<int> _indexStack;
+#endif
+        private List<EmbeddedSignatureData> _embeddedSignatureDataList;
```

Comment:
It compiles, but is using `List<T>` safe for the TypeLoader?

Reply:
Its safe enough.